### ISSUE TITLE
Open Stack Issues

### ISSUE THREAD

**Issue by ks-cajoto** [ks-cspr/network-infra#22]

**Comment by z-caudroy** [comment]
@ks-ialvarez problem with the web interface when we want add a FQDN, contacted support many times and tried their different solutions with @ks-cajoto  but nothing is working.

**Comment by ks-ialvarez** [comment]
apparently the lights went off yesterday in the building and lost this connection again. Moving it back to In progress. Please review @ks-cajoto @z-caudroy

**Comment by ks-cajoto** [comment]
yes, you are right @ks-ialvarez due to cloud team have not just fixed the FDQN issue during VPN creation in Openstack. could you ask them for a fix in terms of time? because we can consider other posibilities to avoid that situation. @z-caudroy
